README.md

```diff
@@ -541,6 +541,69 @@ Example response:<br>
 ```
 
 
+<h2>Esim Vouchers</h2>
+
+`public function esimVouchers(array $vouchers): ?EasyAccess`<br>
+calls `voucher/esim` endpoint of the REST API.
+Full response example can be found here: https://partners-doc.airalo.com/#5a48bb8d-70d1-4030-ad92-4a82eb979281<br>
+```php
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
+use Airalo\Airalo;
+use Airalo\AiraloStatic;
+
+$alo = new Airalo([
+    'client_id' => '<YOUR_API_CLIENT_ID>',
+    'client_secret' => '<YOUR_API_CLIENT_SECRET>',
+]);
+
+
+$vouchers = $alo->esimVouchers([
```

Comment:
is it possible to have the function signature to be more consistent with the rest of the methods, for example quantity is separate parameter for orders, can we do the same here?

Reply:
@MiroIsikiyski basically it's multiple voucher creation so for each esim voucher user can use any quantity, so we cannot add second argument called $quantity. What we can do is to use DTO collection object